docs/usage/reconfigure-renovate.md

```diff
@@ -1,22 +1,41 @@
 ---
 title: Reconfiguring Renovate
-description: How to reconfigure Renovate once it's onboarded
+description: How to reconfigure Renovate after it's onboarded
 ---
 
 # Reconfiguring Renovate
 
-There will likely be times when you need to change your Renovate config.
+There will be times when you need to change your Renovate config.
 There are two recommended approaches:
 
+- Reconfigure via PR
+- Nuke the config and re-onboard
+
 ## Reconfigure via PR
 
-If you wish to make config edits directly, it's recommended to do so via a PR and then run Renovate's config validator to verify it.
-The validator is named `renovate-config-validator` and installed alongside `renovate` itself if you run `npm i -g renovate` or equivalent.
-If it validates your new config then it should be safe to merge.
+If you want to make config edits directly, follow these steps:
+
+1. Create a new Git branch to work on
+1. Install the `renovate` package globally (`npm i -g renovate` or `yarn global add renovate`) to get the `renovate-config-validator` program
+1. Edit your Renovate configuration file
+1. Validate your config by running `insert proper command here`
```

Comment:
I don't know what the proper command is to run the validator (as I've never used it...), so please suggest the proper command. 😄

Reply:
@rarkins Have you seen this comment? We still need to add the proper command. 😄

Reply:
It's `renovate-config-validator`

Reply:
I'll make the commit to fix the command. 😄